### DISCUSSION TITLE
Meet-up Almaty, Kazakhstan: Camunda platform 8 introduction & use-cases presentation

### DISCUSSION THREAD

**Discussion by maria-alcantara** [camunda-community-hub/community#54] [Events and meet-ups]
Hi folks,

I'm very happy to share with you that Vsevolod Martynyuk, recently founded the Camunda Chapter: Almaty. He will host the [first hybrid meet-up in that area!](https://bit.ly/3UmoKBM)

Join him, @NPDeehan, Daniyar Abenov, and Mstislav Martynyuk to learn more not only about Camunda but also about how it is used by local users.

**Info**
:round_pushpin: Hybrid in Almaty
:speech_balloon: Language: Russian and English

👁️ Tagging some community members that could find this interesting @kerlon5 @crowar @alexeybrod @avvensis @liljaylj @imarat88 @SirMullich @muhammed03

🎟️ 👇 
https://bit.ly/3UmoKBM

(![12  Dec 8  Almaty](https://user-images.githubusercontent.com/89025151/205044537-f2dcc7d3-df80-44c9-a2e1-a918c999d7cb.jpg))

**Comment by imarat88** [comment]
I didn't see this notification in time (
Hope that it was a successful meet-up!

**Reply by maria-alcantara** [reply]
Hi @imarat88 , I can tell you from the photos and the comments I've heard that it went really well. You can find the[ recording of the session here](https://community.camunda.com/events/details/camunda-camunda-chapter-almaty-presents-first-ever-official-meetup-in-kazakhstan-camunda-platform-8-introduction-use-cases-presentation/). Maybe next time ;)
